package.json

```diff
@@ -10,6 +10,7 @@
         "test": "test"
     },
     "dependencies": {
+        "@types/node": "^4.2.23",
```

Comment:
I'm not familiar with typescript. Does this need to be declared as a dependency or would it work as a devDependecy as well?